examples/examplesVersionDependency.ttl

```diff
@@ -25,13 +25,13 @@
 	a owl:Ontology ;
 	owl:imports
 		<https://ontologies.exco.com/exeoDiseases> ,
-		<https://w3id.org/semanticarts/ontology/gistCore12.1.0> ,
+		<https://w3id.org/semanticarts/ontology/gistCore> ,
 		<https://w3id.org/semanticarts/ontology/operators> ,
 		<https://w3id.org/semanticarts/ontology/versioning>
 		;
 	skos:definition "A hierarchy of diseases."^^xsd:string ;
 	skos:prefLabel "Disease Taxonomy"^^xsd:string ;
-	skos:scopeNote "The semantic version number for the disease taxonomy is 6.3.0. This version depends on the version of the disease ontology being between 3.1.0 and 4.0.0. It also depends on the version of gist core being between 12.1.0 and 13.0.0."^^xsd:string ;
+	skos:scopeNote "The semantic version number for the disease taxonomy is 6.3.0. This version depends on the version of the imported disease ontology being at least 3.1.0 and less than 4.0.0. It also depends on the version of gist core being at least 12.1.0 and less than 13.0.0."^^xsd:string ;
```

Comment:
Do we really want such a note? Shouldn't the dependency assertions be sufficient? I don't think we should suggest to users that they should include this.

Reply:
Good point, it does not belong here. I will move this to a skos:example on the versioning ontology.

Reply:
I'm confused - you added the example there but it looks like it didn't get deleted here.

Reply:
It is now removed from both from the examples file. I put it in a `skos:example` annotation in the versioning ontology itself.